doc/concepts/wallet.md

```diff
@@ -0,0 +1,35 @@
+
+# Wallets and Wallet Sections
+
+A wallet is a fundamental component within the system that enables users to store and manage their ownership of digital assets, specifically Granular Certificates (GCs) and their corresponding slices. It serves as a centralized repository for users to access, view, and control their assets across multiple registries, issuers, and countries.
+
+## Wallets
```

Comment:
Since the code calls this for "Owner Wallets" should we use the same term here?

Reply:
No, renamed